app/src/common/translations/fr.json

```
@@ -0,0 +1,127 @@
+{
+  "common": {
+    "edit": "Modifier",
+    "save": "Enregistrer",
+    "cancel": "Annuler",
+    "close": "Fermer",
+    "clear": "Vider",
+    "download": "Télécharger",
+    "loading": "Chargement...",
+    "settings": "Paramètres",
+    "help": "Aide",
+    "documentation": "Documentation",
+    "headline": "Entête"
+  },
+  "theme": {
+    "headline": "Thème",
+    "default": "Défaut",
+    "dark": "Sombre"
+  },
+  "language": {
+    "headline": "Langage",
+    "en": "Anglais",
+    "de": "Allemand",
+    "fr": "Français"
```

```suggestion
    "headline": "Langage"
```

As described above, I think it's best to always display the language using the language itself. By removing the translations, the "en.json" will be used as default.